src/App.tsx

```diff
@@ -1,14 +1,65 @@
 /* eslint-disable max-len */
-import React from 'react';
+// eslint-disable-next-line object-curly-newline
+import React, { ChangeEvent, useEffect, useMemo, useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { SortType } from './types/SortType';
+
+const getVisibleTodos = (filterCase: string, todos: Todo[], searchQuery: string) => {
+  const searchedTodo = todos.filter(todo => todo.title.toLowerCase().includes(searchQuery.trim().toLowerCase()));
```

Comment:
I can suggest to add condition like this:
```
if (!!searchQuery.trim()) {
    ...
}
```
Because now you make expensive calculations even then they are no need